docs/rst/conf.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+"""Sphinx configuration."""
```

Comment:
I don't think it should be moved here. Also it doesn't seem related to changing the theme anyway.